docs/getting_started.md

```diff
@@ -0,0 +1,62 @@
+# Running calls through `rtcd`
+
+## Prerequisites
+
+- An [Enterprise](https://docs.mattermost.com/about/editions-and-offerings.html#mattermost-enterprise) licensed Mattermost installation with [Calls](https://github.com/mattermost/mattermost-plugin-calls) installed.
+- A working version of [Docker](https://www.docker.com/) installed.
+
+## Installation
```

Comment:
Can we have two versions of this -- one with docker (as you have) and an alternative "No docker" version? I don't use docker (it's horrible on m1 macs. I had to install it for this, but I'll delete it after). So I just use `make go-run` and it works.

Reply:
We totally can although this guide is really targeting users that want to deploy the service rather than developing it locally.